perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -151,7 +154,8 @@ def RobustRemoteCommand(self, command):
       return self.RemoteCommand(' '.join(wait_command), should_log=False)
     except:
       # In case the error was with the wrapper script itself, print the log.
-      stdout, _ = self.RemoteCommand('cat %s' % wrapper_log, should_log=False)
+      stdout, _ = self.RemoteCommand('cat %s' % wrapper_log,
+                                     should_log=should_log)
```

Comment:
I don't think this should log - if it's interesting it's logged immediately below. Rather, I think the call to "wait_command" on L151 should use the passed in value of `should_log`.